src/main/java/com/openlattice/datastore/services/EdmService.java

```diff
@@ -926,11 +933,16 @@ public void updatePropertyTypeMetadata( UUID propertyTypeId, MetadataUpdate upda
                 .forEach( et -> {
                     List<PropertyType> properties = Lists
                             .newArrayList( propertyTypes.getAll( et.getProperties() ).values() );
-                    entitySetManager.getAllEntitySetsForType( et.getId() )
+                    edmManager.getAllEntitySetsForType( et.getId() )
                             .forEach( es -> eventBus
                                     .post( new PropertyTypesInEntitySetUpdatedEvent( es.getId(), properties ) ) );
                 } );
-        eventBus.post( new PropertyTypeCreatedEvent( getPropertyType( propertyTypeId ) ) );
+
+        PropertyType propertyType = getPropertyType( propertyTypeId );
+        edmManager.createPropertyTypeIfNotExist( propertyType );
```

Comment:
I'm not sure if its checked earlier, but this won't have any effects as the tables will already exists in Postgres. You need to see if it is one the allowed changes and then issue the appropriate ALTER TABLE statements.